source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,59 @@
+.. _pymongo-get-started-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install Dependencies
+
+      Ensure you have Python3 version 3.7 or later,
+      pip, and dnspython installed in your development environment.
+
+      For information about how to install pip, see the
+      `pip Installation guide <https://pip.pypa.io/en/stable/installation/>`__.
+
+      Support for ``mongodb+srv://`` connection URIs requires dnspython.
+      For information about how to install dnspython, see the `dnspython
+      PyPi project page <https://pypi.org/project/dnspython/>`__.
```

Comment:
S: What if this were a list with links? Thinking it might make it clearer + stick to the style guide recs for links

```suggestion
      Ensure you have the following dependencies installed
      in your development environment:
      
       - **Python3 version 3.7 or later.** To learn more about installing Python, see
         `Download Python. <https://www.python.org/downloads/>`__
       - **pip.** For information about how to install pip, see
         `Installation <https://pip.pypa.io/en/stable/installation/>`__ in the pip
         documentation.
         - **dnspython.** Support for ``mongodb+srv://`` connection URIs requires dnspython.
            For information about how to install dnspython, see the `dnspython 
            <https://pypi.org/project/dnspython/>`__ PyPi page.
```

Reply:
Good call!